workshops/views.py

```diff
@@ -554,13 +554,33 @@ def person_bulk_add(request):
                     # the data
                     request.session['bulk-add-people'] = persons_tasks
                     return redirect('person_bulk_add_confirmation')
+        elif form.is_valid() and form.cleaned_data['event']:
+            event = form.cleaned_data['event']
+            try:
+                r = requests.get(event.url + '/api/speaker/')
+                persons_tasks = r.json()
+            except requests.exceptions.HTTPError as e:
+                error_messages.append(
+                    'Request for "{0}" returned status code {1}'
+                    .format(event.url, e.response.status_code)
+                )
+            except (requests.exceptions.ConnectionError,
+                    requests.exceptions.Timeout):
+                error_messages.append("Network connection error.")
+            else:
+                for person in persons_tasks:
+                    person['event'] = event.slug
+                    person['role'] = 'instructor'
```

Comment:
Yes, I think the role should be a field in the API response.  If AMY 
doesn't recognize the value, it should flag it as an error.